Started UtBot Contest, classfileDir = /var/benchmarks/projects/guava/guava/target/classes, classpathString=/var/benchmarks/projects/guava/guava/target/classes:/var/benchmarks/projects/guava/guava/target/guava-26.0-jre.jar:/var/benchmarks/projects/./guava/guava/target/dependency/checker-qual-2.5.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/error_prone_annotations-2.1.3.jar:/var/benchmarks/projects/guava/guava/target/dependency/j2objc-annotations-1.1.jar:/var/benchmarks/projects/guava/guava/target/dependency/jsr305-3.0.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/animal-sniffer-annotations-1.14.jar, outputDir=/home/utbot_mocks/temp/testcases, mocks=Other classes: Mockito
14:06:31.199 | INFO  | Started: warmup: kotlin reflection :: init
14:06:31.283 | INFO  | Finished (in 0.084 sec): warmup: kotlin reflection :: init 
[INIT]
>> [RUN] com.google.common.base.internal.Finalizer 30
14:06:31.294 | DEBUG | -----------------------------------------------------------------------------
14:06:31.295 | INFO  | Contest.runGeneration: Time budget: 30000 ms, Generation timeout=25500 ms, classpath=/var/benchmarks/projects/guava/guava/target/classes:/var/benchmarks/projects/guava/guava/target/guava-26.0-jre.jar:/var/benchmarks/projects/./guava/guava/target/dependency/checker-qual-2.5.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/error_prone_annotations-2.1.3.jar:/var/benchmarks/projects/guava/guava/target/dependency/j2objc-annotations-1.1.jar:/var/benchmarks/projects/guava/guava/target/dependency/jsr305-3.0.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/animal-sniffer-annotations-1.14.jar, methodNameFilter=null
14:06:31.296 | INFO  | ClassUnderTest[ FQN: com.google.common.base.internal.Finalizer
    classfileDir: /var/benchmarks/projects/guava/guava/target/classes
    testClassSimpleName: FinalizerTest
    generatedTestFile: /home/utbot_mocks/temp/testcases/com/google/common/base/internal/FinalizerTest.java
    generatedTestsSourcesDir: /home/utbot_mocks/temp/testcases
]
14:06:31.324 | INFO  | Started: class com.google.common.base.internal.Finalizer
14:06:31.325 | INFO  | Started: preparation class class com.google.common.base.internal.Finalizer: kotlin reflection :: run
14:06:31.357 | INFO  | Finished (in 0.031 sec): preparation class class com.google.common.base.internal.Finalizer: kotlin reflection :: run 
14:06:31.358 | INFO  | Started: 2nd optional soot initialization
14:06:31.359 | INFO  | Finished (in 0.001 sec): 2nd optional soot initialization 
14:06:31.367 | DEBUG |  ... 
14:06:31.374 | INFO  | Started: method com.google.common.base.internal.Finalizer.run()
14:06:31.375 | INFO  |  -- Remaining time budget: 25426 ms, #remaining_methods: 7, budget for method: 3632 ms, solver timeout: 1000 ms, budget for symbolic execution: 2432 ms, budget for concrete execution: 1200 ms,  -- 
14:06:31.703 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.run()1
14:06:33.078 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.run()2
14:06:34.008 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.internal.Finalizer.run()
14:06:34.059 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.run()3
14:06:35.209 | INFO  | (X) Cancelling concrete execution: com.google.common.base.internal.Finalizer.run()
14:06:35.242 | INFO  | Finished (in 3.867 sec): method com.google.common.base.internal.Finalizer.run() 
<StatsForMethod> :
	#generatedTC=3
	WITH NO EXCEPTIONS
14:06:35.291 | DEBUG |  ... 
14:06:35.292 | INFO  | Started: method com.google.common.base.internal.Finalizer.cleanUp(Reference)
14:06:35.292 | INFO  |  -- Remaining time budget: 21504 ms, #remaining_methods: 6, budget for method: 3584 ms, solver timeout: 1000 ms, budget for symbolic execution: 2384 ms, budget for concrete execution: 1200 ms,  -- 
14:06:35.304 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.cleanUp(Reference)1
14:06:35.797 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.cleanUp(Reference)2
14:06:35.951 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.cleanUp(Reference)3
14:06:37.676 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.internal.Finalizer.cleanUp(Reference)
14:06:38.877 | INFO  | (X) Cancelling concrete execution: com.google.common.base.internal.Finalizer.cleanUp(Reference)
14:06:38.930 | INFO  | Finished (in 3.638 sec): method com.google.common.base.internal.Finalizer.cleanUp(Reference) 
<StatsForMethod> :
	#generatedTC=3
	WITH NO EXCEPTIONS
14:06:39.033 | DEBUG |  ... 
14:06:39.034 | INFO  | Started: method com.google.common.base.internal.Finalizer.getFinalizeReferentMethod()
14:06:39.034 | INFO  |  -- Remaining time budget: 17761 ms, #remaining_methods: 5, budget for method: 3552 ms, solver timeout: 1000 ms, budget for symbolic execution: 2352 ms, budget for concrete execution: 1200 ms,  -- 
14:06:39.042 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.getFinalizeReferentMethod()1
14:06:39.292 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.getFinalizeReferentMethod()2
14:06:39.367 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.getFinalizeReferentMethod()3
14:06:41.386 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.internal.Finalizer.getFinalizeReferentMethod()
14:06:42.587 | INFO  | (X) Cancelling concrete execution: com.google.common.base.internal.Finalizer.getFinalizeReferentMethod()
14:06:42.613 | INFO  | Finished (in 3.579 sec): method com.google.common.base.internal.Finalizer.getFinalizeReferentMethod() 
<StatsForMethod> :
	#generatedTC=3
	WITH NO EXCEPTIONS
14:06:42.726 | DEBUG |  ... 
14:06:42.726 | INFO  | Started: method com.google.common.base.internal.Finalizer.startFinalizer(Class, ReferenceQueue, PhantomReference)
14:06:42.727 | INFO  |  -- Remaining time budget: 14069 ms, #remaining_methods: 4, budget for method: 3517 ms, solver timeout: 1000 ms, budget for symbolic execution: 2317 ms, budget for concrete execution: 1200 ms,  -- 
14:06:42.742 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
14:06:42.873 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.startFinalizer(Class, ReferenceQueue, PhantomReference)1
14:06:42.912 | INFO  | Finished (in 0.186 sec): method com.google.common.base.internal.Finalizer.startFinalizer(Class, ReferenceQueue, PhantomReference) 
<StatsForMethod> :
	#generatedTC=1
	FAILED 1 time(s) with 1 different exception(s)"
14:06:42.923 | DEBUG |  ... 
14:06:42.924 | INFO  | Started: method com.google.common.base.internal.Finalizer.getInheritableThreadLocalsField()
14:06:42.924 | INFO  |  -- Remaining time budget: 13872 ms, #remaining_methods: 3, budget for method: 4624 ms, solver timeout: 1000 ms, budget for symbolic execution: 3424 ms, budget for concrete execution: 1200 ms,  -- 
14:06:42.939 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.getInheritableThreadLocalsField()1
14:06:45.820 | ERROR | Symbolic execution FAILED
java.lang.RuntimeException: No method source set for method <java.lang.String: java.lang.String intern()>
	at soot.SootMethod.retrieveActiveBody(SootMethod.java:398) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.ExtensionsKt.jimpleBody(Extensions.kt:136) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.BaseOverriddenWrapper.invoke(CollectionWrappers.kt:89) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.UtBotSymbolicEngine.overrideInvocation(UtBotSymbolicEngine.kt:2663) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.UtBotSymbolicEngine.commonInvokePart(UtBotSymbolicEngine.kt:2399) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.UtBotSymbolicEngine.virtualAndInterfaceInvoke(UtBotSymbolicEngine.kt:2289) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.UtBotSymbolicEngine.invokeResult(UtBotSymbolicEngine.kt:2153) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.UtBotSymbolicEngine.traverseAssignStmt(UtBotSymbolicEngine.kt:884) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.UtBotSymbolicEngine.traverseStmt(UtBotSymbolicEngine.kt:604) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.UtBotSymbolicEngine$traverse$1.invokeSuspend(UtBotSymbolicEngine.kt:464) [utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.UtBotSymbolicEngine$traverse$1.invoke(UtBotSymbolicEngine.kt) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Builders.kt:61) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.flow.AbstractFlow.collect(Flow.kt:212) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.flow.FlowKt__MergeKt$flattenConcat$$inlined$unsafeFlow$1$lambda$1.emit(Collect.kt:134) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.flow.FlowKt__BuildersKt$flowOf$$inlined$unsafeFlow$1.collect(SafeCollector.common.kt:114) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.flow.FlowKt__BuildersKt$flowOf$$inlined$unsafeFlow$1$1.invokeSuspend(Unknown Source) [utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
14:06:46.248 | ERROR | Symbolic execution FAILED ... <<stack trace duplicated>>
14:06:46.348 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.internal.Finalizer.getInheritableThreadLocalsField()
14:06:47.549 | INFO  | (X) Cancelling concrete execution: com.google.common.base.internal.Finalizer.getInheritableThreadLocalsField()
14:06:47.651 | INFO  | Finished (in 4.727 sec): method com.google.common.base.internal.Finalizer.getInheritableThreadLocalsField() 
<StatsForMethod> :
	#generatedTC=1
	FAILED 2 time(s) with 1 different exception(s)"
14:06:47.752 | DEBUG |  ... 
14:06:47.753 | INFO  | Started: method com.google.common.base.internal.Finalizer.getBigThreadConstructor()
14:06:47.753 | INFO  |  -- Remaining time budget: 9042 ms, #remaining_methods: 2, budget for method: 4521 ms, solver timeout: 1000 ms, budget for symbolic execution: 3321 ms, budget for concrete execution: 1200 ms,  -- 
14:06:47.759 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.getBigThreadConstructor()1
14:06:51.074 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.internal.Finalizer.getBigThreadConstructor()
14:06:52.276 | INFO  | (X) Cancelling concrete execution: com.google.common.base.internal.Finalizer.getBigThreadConstructor()
14:06:52.331 | INFO  | Finished (in 4.578 sec): method com.google.common.base.internal.Finalizer.getBigThreadConstructor() 
<StatsForMethod> :
	#generatedTC=1
	WITH NO EXCEPTIONS
14:06:52.492 | DEBUG |  ... 
14:06:52.493 | INFO  | Started: method com.google.common.base.internal.Finalizer.<init>(Class, ReferenceQueue, PhantomReference)
14:06:52.493 | INFO  |  -- Remaining time budget: 4303 ms, #remaining_methods: 1, budget for method: 4303 ms, solver timeout: 1000 ms, budget for symbolic execution: 3103 ms, budget for concrete execution: 1200 ms,  -- 
14:06:52.499 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
14:06:52.789 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.<init>(Class, ReferenceQueue, PhantomReference)1
14:06:52.789 | INFO  | Finished (in 0.296 sec): method com.google.common.base.internal.Finalizer.<init>(Class, ReferenceQueue, PhantomReference) 
<StatsForMethod> :
	#generatedTC=1
	FAILED 1 time(s) with 1 different exception(s)"
14:06:52.803 | INFO  | Started: Flushing tests for [Finalizer] on disk
14:06:53.022 | INFO  | File size for FinalizerTest: 27 KB
14:06:53.024 | INFO  | Finished (in 0.220 sec): Flushing tests for [Finalizer] on disk 
14:06:53.025 | INFO  | Finished (in 21.700 sec): class com.google.common.base.internal.Finalizer 
<StatsForClass> :
	canceled by timeout = false
	#methods = 7, 
	#methods started symbolic exploration = 7
	#methods with at least one TC = 7
	#methods with exceptions = 3
	#generated TC = 13
	#coverage = null
[READY]
